core/crop/structure.py

```diff
@@ -425,4 +439,4 @@ def serialize(self):
         Serialization of the object.
         """
 
-        return {"id": self.id, "username": self.username, "email": self.email}
+        return {"id": self.id, "username": self.username, "email": self.email}
```

Comment:
I think we need to have the new line

Reply:
not sure what that means.

Reply:
This does not show, but there was missing new line at the very end of the Python file. It seems that it is fixed now.